.gitattributes

```diff
@@ -0,0 +1 @@
+*.go text eol=lf
```

Comment:
missing end of line after setting eol to line feed :)

Reply:
No biggie 😆